server/protocols/jmap/src/test/java/org/apache/james/jmap/send/PostDequeueDecoratorTest.java

```diff
@@ -230,4 +239,31 @@ public void doneShouldNotMoveMailFromOutboxToSentWhenInvalidMessageIdProvided()
         MessageResultIterator resultIterator = mailbox.getMessages(MessageRange.one(UID), FetchGroupImpl.FULL_CONTENT, mailboxSession);
         assertThat(resultIterator).hasSize(1);
     }
+
+    @Test
+    public void doneShouldCopyMailFromOutboxToSentOnlyOneTimeWhenSuccess() throws Exception {
```

Comment:
It emphasizes on the copy action when success, not the success action on email because basically that mail can be two times success if user would like to send it again. Then that name can be fine :). Could we accept that?